src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -1,8 +1,41 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+
+import { ConfigurationService } from '~/app/shared/api/configuration.service';
+import { OrchestratorService } from '~/app/shared/api/orchestrator.service';
+import { SummaryService } from '~/app/shared/services/summary.service';
 
 @Component({
   selector: 'cd-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.scss']
 })
-export class DashboardComponent {}
+export class DashboardComponent implements OnInit {
+  detailsCardData: any = {};
+  // function for keyvalue order
+  zero: any = () => 0;
+
+  constructor(
+    private summaryService: SummaryService,
+    private configService: ConfigurationService,
+    private orchService: OrchestratorService
+  ) {}
+
+  ngOnInit() {
+    this.getDetailsCardData();
+  }
+
+  getDetailsCardData() {
+    this.configService.get('fsid').subscribe((data) => {
+      this.detailsCardData['FSID'] = data['value'][0]['value'];
+    });
+    this.orchService.status().subscribe((data) => {
```

Comment:
@Pegonzal this call already gives the orchestrator backend.
```suggestion
    this.mgrModuleService.getConfig('orchestrator').subscribe((data) => {
```

Reply:
Oh my... I searched for one but couldn't find any.... Thanks for letting me know, I'll change it

Reply:
Now using the correct service so no need to modify the backend as I was previously doing